python/tvm/relay/op/contrib/__init__.py

```diff
@@ -15,7 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 # pylint: disable=wildcard-import
-"""Neural network related operators."""
-from __future__ import absolute_import as _abs
-from .contrib import *
+"""Contrib modules."""
```

Comment:
I feel this is a bad place to put dnnl in the contrib. Or maybe we should move the original `contrib` to another namespace.

Reply:
The reason we put here is that all functions under `dnnl` are adding a specific target attribute to Relay ops. Maybe we can make a `relay/op/annotation` folder and put them there if you have a concen?